Client/mods/deathmatch/logic/luadefs/CLuaTimerDefs.cpp

```diff
@@ -111,6 +113,28 @@ int CLuaTimerDefs::KillTimer(lua_State* luaVM)
     return 1;
 }
 
+bool CLuaTimerDefs::IsTimerPaused(lua_State* luaVM, CLuaTimer* pLuaTimer)
+{
+    //  bool isTimerPaused ( timer theTimer )
+    CLuaMain* luaMain = m_pLuaManager->GetVirtualMachine(luaVM);
+    if (!luaMain)
+        return false;
```

Comment:
this is pointless. Just as the `lua_State*` parameter.
Add `noexcept` and dont use hungarian notation (just `timer` is sufficient as parameter name)